hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationStats.java

```diff
@@ -44,33 +45,83 @@
  */
 public class MigrationStats {
 
+    /**
+     * In {@link TimeUnit#MILLISECONDS}
+     *
+     * @see #getLastRepartitionTime()
+     */
     @Probe(name = MIGRATION_METRIC_LAST_REPARTITION_TIME, unit = MS)
     private final AtomicLong lastRepartitionTime = new AtomicLong();
 
+    /**
+     * In {@link TimeUnit#NANOSECONDS}
+     *
+     * @see #getPlannedMigrations()
+     */
```

Comment:
This value is not a measurement of time, it's a counter for total operations; this java doc is not necessary.

Reply:
All `MeasureUnit` Javadocs removed.